src/components/App.jsx

```diff
@@ -44,16 +44,17 @@ export class App extends Component {
             <Route path="/connectedDevices/:deviceId" component={Devices} />
             <Route path="/sessions" component={Sessions} />
             <Route path="/storage" component={Storage} />
-            <Route exact path="/permissions" component={Permissions} />
+            <Route exact path="/permissions/apps" component={PermissionsTab} />
             <Route
               exact
-              path="/permissions/:app"
+              path="/permissions/apps/:app"
```

Comment:
if the route will also be used for konnector, please don't use the word `apps`. `slug` or something similar will be better